lib/tasks/carriers.rake

```diff
@@ -0,0 +1,8 @@
+namespace :carriers do
```

Comment:
Same as above, awesome idea if we had production customers, but since we don't I think it would be easier to stick this `backfilling` inside the migration itself